docs/Versioning.md

```diff
@@ -26,6 +26,7 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [Windows ML Availability](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
 |------------------------------|--------------------|--------------------|----------------------|------------------|------------------|
+| 1.10.0 | **1.10** down to 1.2 | 14 | 2 | 7 | Windows AI 1.10+ |
```

Comment:
Not sure if this line is correct

Reply:
It's fine. It might be updated later if newer version of onnx comes out.